orchard-core/src/main/scala/com/salesforce/mce/orchard/util/Retry.scala

```diff
@@ -0,0 +1,23 @@
+package com.salesforce.mce.orchard.util
+
+import com.krux.stubborn.RetryDefaults
+import com.krux.stubborn.policy.Policy
+
+import scala.util.{Failure, Try}
+
+object Retry extends RetryDefaults {
+
+  @annotation.tailrec
+  def apply[T](numTries: Int = defaultMaxRetry, policy: Policy = defaultPolicy)(
```

Comment:
you can just use this one, no? https://github.com/salesforce/stubborn/blob/master/src/main/scala/com/krux/stubborn/Retryable.scala

Reply:
Yes, you are right. I will use Retryable.retry.

Reply:
there's nothing stops you from writing a function that uses `Retryable` to return `Try`/`Either`.

Reply:
That is correct.

Reply:
Please  review this stubborn PR, which adds a method to wrap Retryable.retry() in a Try:
https://github.com/salesforce/stubborn/pull/4

Reply:
you don't have to change stubborn. have a method that wraps try here.

Reply:
I was thinking this retry in a Try wrapper idiom would be common across projects. 

Sure, I can add a method in this project to wrap a Try around retry().

Reply:
I crated a method to wrap Retryable.retry in a Try in the latest commit.  Please let me know what you think.